### PR TITLE
Drop support for php 7.0 on next major

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: config/projects.yml [hunk 1]
```diff
@@ -182,7 +182,7 @@ doctrine-phpcr-admin-bundle:
   branches:
     master:
       # sonata 2 should still support php 5.6 to be consistent with the symfony cmf
-      php: ['5.6', '7.0', '7.1']
+      php: ['5.6', '7.1']
```
Comment (greg0ire) [inline config/projects.yml]: @dbu @lsmith77 please confirm you are ok with this

File: config/projects.yml [hunk 2]
```diff
@@ -257,7 +257,7 @@ google-authenticator:
   docs_target: false
   branches:
     master:
-      php: ['7.0', '7.1']
+      php: ['7.1']
     2.x:
       php: ['7.0', '7.1']
```
Comment (greg0ire) [inline config/projects.yml]: Too late it is released

File: config/projects.yml (hunk 1 above)
Comment (dbu) [inline config/projects.yml]: we are about to release 2.0, so i think its too early to drop 7.0. also dropping 7.0 but not 5.6 would be rather pointless. i suggest we wait until we tagged the stable release and created a 2.x branch and only then drop 5.6 and 7.0 for master that will be 3.x

Comment (greg0ire) [inline config/projects.yml]: Ok I'll revert this change :+1:

**Comment by greg0ire** [comment]
> what is the time horizon of the next majors? be aware that according to http://php.net/supported-versions.php php 7.0 is still actively supported until end of this year and security fixes until next year...

Since there is a lot of inter-dependencies, and a lot of package, the releases will not be synchronized (I think it is for the better). For instance, google authenticator 2 is already released, which, by the way, means there already is an exception to the "drop php 7.0" proposal

**Comment by greg0ire** [comment]
> be aware that according to http://php.net/supported-versions.php php 7.0 is still actively supported until end of this year and security fixes until next year...

Since it's a next major, I personally think we should start by supporting only the latest versions of our deps, especially this one. We're not hurting anyone by doing that, and since we release at a very slow pace (at least that's how I see it), we shouldn't choose to get stuck with 7.0 for a potentially long period of time.

**Comment by Soullivaneuh** [comment]
Rest in Peace, PHP 7.0.

**Pull request merged**